### ISSUE TITLE
Upgrade to v8.7.2 breaks login

### ISSUE THREAD

**Issue by edvasqueza** [directus/docker#122]
Hello, I'm running the docker version (from dockerhub) and today I upgraded from v8.7.1-apache to v8.7.2-apache. After the upgrade the login page shows "Couldn't reach API" when correct credentials are given.

Besides that I could notice a difference in the GET /projectid endpoint

v8.7.1
```{"data":{"api":{"version":"8.7.1","requires2FA":false,"database":"mysql","project_logo":null,"project_color":"#263238","project_foreground":null,"project_background":null,"project_public_note":null,"default_locale":null,"telemetry":true,"project_name":"Project 1"},"server":{"max_upload_size":2097152,"general":{"php_version":"7.3.18","php_api":"apache2handler"}}}}```

v8.7.2
```{"data":{"api":{"requires2FA":false,"project_logo":null,"project_color":"#263238","project_foreground":null,"project_background":null,"project_public_note":null,"default_locale":null,"telemetry":true,"project_name":"Project 1"}},"public":true}```

Did I miss any configuration needed to upgrade?

**Comment by benhaynes** [comment]
@directus/team — this seems to be the HTTPS issue, right? Is there a way we can get Docker to run properly out-of-the-box?

**Comment by rijkvanzanten** [comment]
@edvasqueza Ensure that your API is running under HTTPS.

@WoLfulus is that something we can provide ootb?

**Comment by mvhirsch** [comment]
I'm running into same issue. I'm using Directus on Kubernetes, so the APIs request is probably HTTP, but for public it is HTTPS (using a L7 Ingress, which does SSL-handshaking for public).

Is that my issue? Do I need to pass-through SSL to Directus directly?

**Comment by edvasqueza** [comment]
> I'm running into same issue. I'm using Directus on Kubernetes, so the APIs request is probably HTTP, but for public it is HTTPS (using a L7 Ingress, which does SSL-handshaking for public).
> 
> Is that my issue? Do I need to pass-through SSL to Directus directly?

I'm also running it on kubernetes. Did a rollback to v8.7.1 in the meantime.

**Comment by WoLfulus** [comment]
I've updated the examples to quickly enable https through caddy.

As for kubernetes, if the api requests are being made through `https` and `app` served on it as well, and under the same domain, there shouldn't be a problem.

Please let me know if that's the case.

**Comment by jroxendal** [comment]
From the Caddy example I get:
```
root@lit:~/directus-docker/examples/basic# curl https://localhost:8443
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.```

**Comment by enodr** [comment]
Hi same problem here, installed directus docker per official instructions. Everything went fine but when trying to login there is an error "Couldn't reach API". The doc mentions HTTPS being mendatory but nothing specific as to what and where to look. Shouldn't the default config work out of the box ?

**Comment by jroxendal** [comment]
I couldn't get this to work with the caddy in docker-compose, but installing caddy locally and reverse-proxying to Directus worked. On a mac:

First, expose the api to port 8080 by adding 
```
ports:
    - 8080:80
```
to your docker-compose.yml under the directus block. Then, `brew install caddy`

```
localhost:2020 {
  bind {$ADDRESS}
  reverse_proxy localhost:8080
  tls internal
}
```

This makes https://localhost:2020 point to Directus, which now works to log in to.

**Comment by rijkvanzanten** [comment]
This whole "HTTPS is required for cross site cookies" situation is super annoying :/ Here's to hoping browser makers don't decide to do another stunt like that soon..

In the meantime, I've released v8.8.0 that makes the cookie's Secure flag configurable and disabled by default, so that should take care of all these local running issues 👍

**Comment by spawnrider** [comment]
> In the meantime, I've released v8.8.0 that makes the cookie's Secure flag configurable and disabled by default, so that should take care of all these local running issues 👍

This is configurable but not using Environment variables :( Is it possible to add some variable to set cookie settings? 
For exemple : DIRECTUS_COOKIE_SECURE & DIRECTUS_COOKIE_SAME_SITE
